frontend/src/metabase/components/Schedule/strings.ts

```diff
@@ -64,6 +64,7 @@ export const getScheduleStrings = () => {
     daily: c("adverb").t`daily`,
     weekly: c("adverb").t`weekly`,
     monthly: c("adverb").t`monthly`,
+    cron: c("adverb").t`custom`,
```

Comment:
What context does "custom" appear in? This caught my eye because I don't think it can be an adverb (except in rare unrelated contexts).

Reply:
I thought about it and it seemed like adverb is more appropriate than adjective here, although I may read it incorrectly. Here's an example:

<img width="516" alt="image" src="https://github.com/user-attachments/assets/6dcf5d4b-fada-4455-ae0c-08743c6db294" />

Reply:
I'd just remove the context string from "custom". The strings "hourly", "monthly" etc need a context string because they can be adjectives (as in "monthly rate") or adverbs ("send monthly). In fact it occurs to me that we don't even need a context string for "by the minute", so feel free to remove that too.